doc/visual-programming/source/widgets/evaluation/testandscore.rst

```diff
@@ -52,10 +52,14 @@ than one widget to test multiple learners with the same procedures.
       we select *Separate Test Data* signal in the communication channel
       and select Test on test data.
 
-2. Only *Test on test data* requires a target class, e.g. having the
-   disease or being of subvariety *Iris setosa*. When *Target class* is
-   (None), the methods return the average value. Target class can be
-   selected at the bottom of the widget. 
+2. For classification, can *Target class* can be selected at the bottom of the widget.
```

Comment:
Erm, sorry but now there is one `can` too many.

Reply:
fixed